### DISCUSSION TITLE
Welcome to AsyncHTTPSRequest_Generic Discussions!

### DISCUSSION THREAD

**Discussion by khoih-prog** [khoih-prog/AsyncHTTPSRequest_Generic#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by glennswest** [comment]
Im looking at a way of downloading arbitrary length files and processing the download in chunks.
ie, as the download comes in, want to write to sd.
For example: download of new firmware/flash contents, or download of web content including images.
It will be served back out similarly.
Short scanning of code I see mention of chunks, but dont see a way of "streaming" the results out via callback?

**Reply by glennswest** [reply]
Hmm, looks like the onData might be the way.

**Reply by glennswest** [reply]
In looking at the chunk code, I notice it looking for \n\r, Im wondering the intent here.
